
 gtk2hs doesn't export Event anymore, presumably because it wants us to use
 the EventM monad. Patch sent upstream.

  -- Chris Lamb <lamby@debian.org>  Sun, 08 Mar 2009 03:49:54 +0000

--- a/Bustle.hs	2009-03-05 22:29:48.000000000 +0000
+++ b/Bustle.hs	2009-03-05 22:29:52.000000000 +0000
@@ -30,6 +30,7 @@
 import Bustle.Diagram
 
 import Graphics.UI.Gtk
+import Graphics.UI.Gtk.Gdk.Events
 import Graphics.Rendering.Cairo
 
 import System.Environment (getArgs)
